plugins/vpc-bridge/config/netconfig.go

```diff
@@ -59,6 +63,21 @@ type netConfigJSON struct {
 	ServiceCIDR      string   `json:"serviceCIDR"`
 }
 
+// RuntimeConfig are the runtime options which will be populated dynamically by the runtime
+// based on the requested capability.
+// https://www.cni.dev/docs/conventions/#dynamic-plugin-specific-fields-capabilities--runtime-configuration
+type RuntimeConfig struct {
+	PortMappings []PortMapping `json:"portMappings,omitempty"`
+}
+
+// PortMapping contains the container port to host port mapping information.
+type PortMapping struct {
+	HostPort      int    `json:"hostPort"`
+	ContainerPort int    `json:"containerPort"`
+	Protocol      string `json:"protocol"`
+	HostIP        string `json:"hostIP,omitempty"`
```

Comment:
Where is "HostIP" used? I don't see any references to it.

Reply:
`HostIP` is part of the Pod spec. Reference [here](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#ports).

So we can use this field to specify the VIP on host to which we want to bind the port to. Reference [here](https://github.com/microsoft/hcsshim/blob/4f8d26f7d627ffa68809c4f695c53bb6673d0fd1/hcn/hcnpolicy.go#L95).

In the present PR I have not used the same since the implicit VIP is the host instance IP. Should we add support for passing HostIP?

Reply:
We can add it later if we need to.
Also please make Protocol the first field.

Reply:
Yes sure. I will then omit this field from the config for the time being.